code/lib/manager-api/src/modules/refs.ts

```diff
@@ -75,8 +73,10 @@ export const getSourceType = (source: string, refId?: string) => {
   const { origin: localOrigin, pathname: localPathname } = location;
   const { origin: sourceOrigin, pathname: sourcePathname } = new URL(source);
 
-  const localFull = `${localOrigin + localPathname}`.replace(findFilename, '');
-  const sourceFull = `${sourceOrigin + sourcePathname}`.replace(findFilename, '');
+  const localFull = `${localOrigin + localPathname}`.replace('/iframe.html', '').replace(/\/$/, '');
+  const sourceFull = `${sourceOrigin + sourcePathname}`
+    .replace('/iframe.html', '')
+    .replace(/\/$/, '');
```

Comment:
The findFilename regex was complex, and wasn't working well.

Composed storybooks will always look for `iframe.html`, so there's no need for this to be so dynamic.